packages/react-server/src/ReactFlightServer.js

```diff
@@ -102,14 +108,12 @@ export type Request = {
   writtenSymbols: Map<Symbol, number>,
   writtenModules: Map<ModuleKey, number>,
   writtenProviders: Map<string, number>,
+  identifierPrefix?: string,
```

Comment:
We should never have optional properties in internal data structures since we want consistent hidden classes. Only short lived and rare config objects and such should have those.

This should just be `string` always and set to empty string when you initialize it if no option is passed.